### PR TITLE
[VEN-2758]: LST ETH pool on BNB chain

### PR DESCRIPTION
**Pull request by kkirka**
This PR contains a VIP with the commands required to add LST ETH pool to BNB chain

### CODE REVIEW

**Review by chechu — COMMENTED** [review]
I would add `setPrime` on the new comptroller contracts, to keep them ready to enable any market as Prime

File: vips/vip-400/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+export const WEETH_ONEJUMP_REDSTONE_ORACLE = "0xb661102c399630420A4B9fa0a5cF57161e5452F5";
+export const WEETH_ONEJUMP_CHAINLINK_ORACLE = "0x3b3241698692906310A65ACA199701843404E175";
+
+export const CHAINLINK_ORACLE = "0x1B2103441A0A108daD8848D8F5d790e4D402921F";
+export const WSTETH_CHAINLINK_FEED = ""; // TBD
+export const WEETH_CHAINLINK_FEED = "0xF37Be32598E9851f785acA86c2162e7C1A8466dd";
+export const CHAINLINK_STALE_PERIOD = 26 * 60 * 60; // 26 hours
+
+export const REDSTONE_ORACLE = "0x8455EFA4D7Ff63b8BFD96AdD889483Ea7d39B70a";
+export const WSTETH_REDSTONE_FEED = "0x9b2C948dbA5952A1f5Ab6fA16101c1392b8da1ab"; // TODO: use "0xF43a01A39Fd28950031a42861faD703147d5C8e7"
```
Comment (chechu) [inline vips/vip-400/bscmainnet.ts:39]: Use `0xa76dB2Cb356ba111cCB5a7Ca369D17E1592f42Dd`, the wstETH/stETH price feed from RedStone

Screenshot 2024-09-11 at 15 28 34

File: vips/vip-400/bsctestnet.ts [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+        symbol: "weETH",
+      },
+      ...commonSpec,
+    },
+    riskParameters: {
+      collateralFactor: parseUnits("0.9", 18),
+      liquidationThreshold: parseUnits("0.93", 18),
+      supplyCap: parseUnits("125", 18),
+      borrowCap: parseUnits("65", 18),
+      reserveFactor: parseUnits("0.2", 18),
```
Comment (chechu) [inline vips/vip-400/bsctestnet.ts:99]:
```suggestion
      supplyCap: parseUnits("400", 18),
      borrowCap: parseUnits("200", 18),
      reserveFactor: parseUnits("0.25", 18),
```

File: vips/vip-400/bsctestnet.ts [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+      protocolSeizeShare: parseUnits("0.01", 18),
+    },
+    initialSupply: {
+      amount: parseUnits("2", 18),
```
Comment (chechu) [inline vips/vip-400/bsctestnet.ts:103]:
```suggestion
      amount: parseUnits("4.43236349753311919", 18),
```

File: vips/vip-400/bsctestnet.ts [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+        decimals: 18,
+        symbol: "ETH",
+      },
+      ...commonSpec,
+    },
+    riskParameters: {
+      collateralFactor: parseUnits("0.9", 18),
+      liquidationThreshold: parseUnits("0.93", 18),
+      supplyCap: parseUnits("125", 18),
+      borrowCap: parseUnits("110", 18),
```
Comment (chechu) [inline vips/vip-400/bsctestnet.ts:130]:
```suggestion
      collateralFactor: parseUnits("0", 18),
      liquidationThreshold: parseUnits("0", 18),
      supplyCap: parseUnits("450", 18),
      borrowCap: parseUnits("400", 18),
```

File: vips/vip-400/bscmainnet.ts [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[WEETH, WEETH_REDSTONE_FEED, redstoneStalePeriod]],
+      },
+      (() => {
+        // wstETH price feed reverts in the simulation environment due to staleness check,
+        // so we have to use a stub value for testing
+        if (overrides.hardcodeWstETHPrice) {
+          return {
+            target: REDSTONE_ORACLE,
+            signature: "setDirectPrice(address,uint256)",
```
Comment (chechu) [inline vips/vip-400/bscmainnet.ts:211]: Could we set this in the simulation?

File: vips/vip-400/bscmainnet.ts [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+        signature: "setValidateConfig((address,uint256,uint256))",
+        params: [[token, UPPER_BOUND_RATIO, LOWER_BOUND_RATIO]],
+      })),
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            WSTETH,
+            [WSTETH_ONEJUMP_REDSTONE_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
```
Comment (chechu) [inline vips/vip-400/bscmainnet.ts:233]:
```suggestion
            [WSTETH_ONEJUMP_CHAINLINK_ORACLE, WSTETH_ONEJUMP_REDSTONE_ORACLE, WSTETH_ONEJUMP_REDSTONE_ORACLE],
```

File: vips/vip-400/bscmainnet.ts [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+            params: [WSTETH, FIXED_LST_PRICE],
+          };
+        } else {
+          return {
+            target: REDSTONE_ORACLE,
+            signature: "setTokenConfig((address,address,uint256))",
+            params: [[WSTETH, WSTETH_REDSTONE_FEED, redstoneStalePeriod]],
+          };
+        }
+      })(),
```
Comment (chechu) [inline vips/vip-400/bscmainnet.ts:221]: We should also add the configuration of the wstETH feed in the ChainlinkOracle: `0x4c75d01cfa4D998770b399246400a6dc40FB9645`

Comment (kkirka) [reply vips/vip-400/bscmainnet.ts]: [a38fa42](https://github.com/VenusProtocol/vips/pull/373/commits/a38fa42eecbe361e565f23c248850fa859759ecb)

File: vips/vip-400/bscmainnet.ts (hunk 6 above)
Comment (kkirka) [reply vips/vip-400/bscmainnet.ts]: [a38fa42](https://github.com/VenusProtocol/vips/pull/373/commits/a38fa42eecbe361e565f23c248850fa859759ecb)

File: vips/vip-400/bscmainnet.ts (hunk 5 above)
Comment (kkirka) [reply vips/vip-400/bscmainnet.ts]: [a38fa42](https://github.com/VenusProtocol/vips/pull/373/commits/a38fa42eecbe361e565f23c248850fa859759ecb)

File: vips/vip-400/bscmainnet.ts [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+          target: vToken.address,
+          signature: "setProtocolSeizeShare(uint256)",
+          params: [riskParameters.protocolSeizeShare],
+        },
+      ]),
+
+      // Conversions Config
+      ...Object.entries(converterBaseAssets).map(([converter, baseAsset]: [string, string]) => {
+        const tokens = [WSTETH, WEETH];
+        const conversionConfigs = tokens.map(() => [0, ConversionAccessibility.ALL]);
```
Comment (chechu) [inline vips/vip-400/bscmainnet.ts:321]: I would set 1e14 for the conversion incentives here, as it was made in the [VIP-350](https://app.venus.io/#/governance/proposal/350?chainId=56) for the rest of the markets on BNB Chain, following the Chaos labs recommendations

Comment (chechu) [reply vips/vip-400/bscmainnet.ts]: https://github.com/VenusProtocol/vips/pull/373/commits/0adbbd1173b83e29af85e2d7d5a00373a047682e

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/370?chainId=56

**Pull request merged**